src/tests/Directory.Build.props

```diff
@@ -178,8 +178,8 @@
 
   <!-- Set Test Wrapper running host OS -->
   <PropertyGroup>
-    <TestWrapperTargetsWindows>false</TestWrapperTargetsWindows>
-    <TestWrapperTargetsWindows Condition=" ('$(TargetsWindows)' != '' And '$(TargetsWindows)' ) OR ('$(TargetOS)' == 'android' And '$(TargetArchitecture)' == 'arm64' ) OR ($(WindowsHost) And '$(TargetArchitecture)' == 'wasm')">true</TestWrapperTargetsWindows>
+    <TestWrapperTargetsWindows Condition="'$(TestWrapperTargetsWindows)' == ''">false</TestWrapperTargetsWindows>
+    <TestWrapperTargetsWindows Condition=" ('$(TargetsWindows)' != '' And '$(TargetsWindows)' ) OR ('$(TargetOS)' == 'android' And '$(TargetArchitecture)' == 'arm64' )">true</TestWrapperTargetsWindows>
```

Comment:
It is not clear to me any changes to `TestWrapperTargetsWindows` logic are required.

Here's how the upstream condition will evaluate on Windows-targeting-WASM:
- ` ('$(TargetsWindows)' != '' And '$(TargetsWindows)' ) OR ('$(TargetOS)' == 'android' And '$(TargetArchitecture)' == 'arm64' ) OR ($(WindowsHost) And '$(TargetArchitecture)' == 'wasm')` => `true` due to `(($(WindowsHost) And '$(TargetArchitecture)' == 'wasm')`.
On Not-Windows-targeting-WASM:
- => `""` (none of the Windows conditions are `true`).

Which is how it is supposed to be.

Reply:
You are right, it's not required. Thanks.